sonic_api.yaml

```diff
@@ -1666,6 +1666,57 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# Static Route Expiry
+#----------------------------------------------
+  '/config/vrf/static_route/route_expiry':
+    get:
+      operationId: ConfigVrfStaticRouteExpiryGet
+      summary: API for caller to get the custom static route expiry time
+      description: Returns a JSON response with the expiration time for all non-persistent static routes
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/StaticRouteExpiryTime'
+        '401':
+          description: Invalid authentication credentials
+          schema:
+            $ref: '#/definitions/Error'
+        '500':
+          description: Internal service error
+          schema:
+            $ref: '#/definitions/Error'
+        '503':
+          description: Maintanence mode
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: API for caller to set the custom static route expiry time
+      operationId: ConfigVrfStaticRouteExpiryPost
+      description: Sets the custom expiration time for all non-persistent static routes
```

Comment:
Can you please give more details on how the behavior would be when time is changed. Also specify the excpection cases like wb

Reply:
I think we should not out all those details in this `yaml` file, but rather in the design document.

Reply:
Currently the yml file has 'some' details on how the API behaves. I suggest lets put some details so that its easy to share and with partner teams.